lms/envs/common.py

```diff
@@ -346,7 +346,7 @@
     'ENABLE_OPENBADGES': False,
 
     # Enable LTI Provider feature.
-    'ENABLE_LTI_PROVIDER': False,
+    'ENABLE_LTI_PROVIDER': True,
```

Comment:
Please don't change this value. It is default edx settings that should not be changed. Any values from this config could be overwritten in the `lms.env.json`/`cms.env.json` files